src/website/app/Router.js

```diff
@@ -118,10 +118,6 @@ export default function Router({ demoRoutes }: Props) {
           // $FlowFixMe
           const selectedDemo = flatDemoRoutes[componentId];
 
-          if (selectedDemo.redirect) {
-            return <Redirect to={`/components/${selectedDemo.redirect}`} />;
```

Comment:
This one allows a demo to include a `redirect` property that defines which demo to redirect to.

Reply:
At the time of this comment, the Layout route uses this functionality.